test/controllers/ChargeSummaryControllerSpec.scala

```diff
@@ -60,6 +60,130 @@ class ChargeSummaryControllerSpec extends ChargeSummaryControllerHelper {
           testSupportingAgentDeniedAccess(action(id1040000123))(fakeRequest)
         } else {
           "render the charge summary page" when {
+            "charge history & your self assessment charges feature switch is enabled and there is a user" that {
+              "provided with an id associated to a POA1 Debit" in new Setup(
+                financialDetailsModelWithPoaOneAndTwo()) {
+                enable(ReviewAndReconcilePoa)
```

Comment:
would like nice to have a enable method where we can pass variable number of argument, as this would be a step forward I think